connectors/sources/mysql.py

```diff
@@ -387,9 +413,9 @@ async def _connect(self, query, fetch_many=False, **query_kwargs):
                     retry += 1
 
     async def fetch_all_tables(self):
-        return await anext(self._connect(query=QUERIES["ALL_TABLE"]))
+        return await anext(self._connect(query=self.queries.all_tables()))
 
-    async def fetch_rows_for_table(self, table=None, query=QUERIES["TABLE_DATA"]):
+    async def fetch_rows_for_table(self, table=None, query=None):
```

Comment:
Question: is there any impact changing the default value for `query`?

Reply:
Note that I'm aware of. We call this method at two places: 
- https://github.com/elastic/connectors-python/pull/675/files#diff-996ac0843d7656c71035ae294db22a975939f9fd73685747f5faa8e3b6d64d4eR523
- https://github.com/elastic/connectors-python/pull/675/files#diff-996ac0843d7656c71035ae294db22a975939f9fd73685747f5faa8e3b6d64d4eR446

At both places a query is present. This will also be restructured afterwards.